### DISCUSSION TITLE
C-TAEA and how it works related to restrictions

### DISCUSSION THREAD

**Discussion by RafaHPSUnicamp** [community/community#77998] [Programming Help]
### Select Topic Area

Question

### Body

Hi, I am revising my work related to C-TAEA and other algoritms. So, it is really common that C-TAEA can select points to enter in Pareto Front that disobey certain restrictions? For example, my C-TAEA have points which variables have values lower as their restrictions supposed to not allow. It is normal?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @RafaHPSUnicamp ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
